src/polyswarmd/eth.py

```diff
@@ -151,6 +160,33 @@ def build_transaction(call, nonce):
     return call.buildTransaction(options)
 
 
+def is_withdrawal(tx):
+    """
+    Take a transaction and return True if that transaction is a withdrawal
+    """
+    error = True
+    function_hash = g.chain.w3.toHex(tx.data[:4])
```

Comment:
Earlier in the file we use HexBytes(tx.data) for parsing out bytes from tx data, think we should use the same method in both places